pkg/manage/cmd_config.go

```diff
@@ -0,0 +1,151 @@
+package manage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/OpenSlides/openslides-manage-service/pkg/datastore"
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+)
+
+const configHelp = `Get or set config values
+
+This command gets or sets the config values for an organisation.
+
+Example:
+
+$ manage config voting
+disabled
+$ manage config set voting enabled
+$ manage config voting
+enabled
+`
+
+// CmdConfig initializes the config command.
+func CmdConfig(cfg *ClientConfig) *cobra.Command {
+	values := []string{
+		"voting",
```

Comment:
Please add a comment here that at the moment this code only supports boolean fields.

Reply:
That is not true. It only supports the voting key, which by accident is boolean. You can not add other boolean values without changing the server anyway.